style.css

```diff
@@ -497,6 +524,10 @@ hr {
 }
 
 @media screen and (min-width: 768px) {
+  .mobile-header {
```

Comment:
Good work implementing the redirections for each section on the webpage! 💯 

- Kindly, as is in the requirements, apply the JS code necessary to show and hide the menu when a user clicks on the hamburger button

- Kindly, create the "X" button to close the popup menu as in the Figma template


![image](https://user-images.githubusercontent.com/93630700/223138581-33f95a23-d683-4ad0-ac4b-0250a08ebb7e.png)